tests/components/minecraft_server/test_binary_sensor.py

```diff
@@ -22,16 +22,27 @@
 
 
 @pytest.mark.parametrize(
-    ("mock_config_entry", "server", "status_response"),
+    ("mock_config_entry", "server", "lookup_function_name", "status_response"),
     [
-        ("java_mock_config_entry", JavaServer, TEST_JAVA_STATUS_RESPONSE),
-        ("bedrock_mock_config_entry", BedrockServer, TEST_BEDROCK_STATUS_RESPONSE),
+        (
+            "java_mock_config_entry",
+            JavaServer,
```

Comment:
If server is only used for .__name__ then you could combine the server and lookup function name.

Reply:
It is used. In this example please check line 56:
`return_value=server(host=TEST_HOST, port=TEST_PORT),`

I could remove the `lookup_function_name` parametrize and instead derive the corresponding function name from the server type in an if else inside the test function. But I thought with parametrize it's nicer. What do you think?

Reply:
yes, sounds good as is.